events/admin.py

```diff
@@ -61,9 +61,10 @@ class KeywordAdmin(AutoIdBaseAdmin, TranslationAdmin, VersionAdmin):
 admin.site.register(Keyword, KeywordAdmin)
 
 
-class KeywordSetAdmin(BaseAdmin):
-    fields = ('data_source', 'origin_id', 'name', 'keywords', 'usage')
+class KeywordSetAdmin(AutoIdBaseAdmin):
+    fields = ('id', 'data_source', 'origin_id', 'name', 'keywords', 'usage')
     autocomplete_fields = ('keywords', )
+    readonly_fields = ('id',)
```

Comment:
I believe we need to do the same as with data sources and languages here: allow specifying the id on create, but not otherwise. In my understanding, if the user sets an id here, `AutoIdBaseAdmin` will use that to generate a `data_source:id` format id?

Reply:
We discussed this on Slack and decided to modify `AutoIdBaseAdmin` so it uses origin_id to create id of form `data_source:origin_id` if origin_id has been set.